Build issue - Duplicate identifier

I tried to use current build, but faced error:

`../jsonld-dataset-proxy/dist/setProxy/SetProxy.d.ts(18,3): error TS2300: Duplicate identifier '[_proxyContext]'.`

It affects `@ldo/jsonld-dataset-proxy v1.0.0-alpha.38`.

This is the offending snippet from build:

```ts
declare abstract class SetProxy<T extends NonNullable<RawValue> = NonNullable<RawValue>> extends BasicLdSet<T> {
  [_getUnderlyingDataset]: Dataset<Quad, Quad>;
  [_getUnderlyingMatch]: QuadMatch;
  [_proxyContext]: ProxyContext;
  [_proxyContext]: ProxyContext; // <== THIS IS THE ISSUE
  [_writeGraphs]: GraphNode[];
  [_isSubjectOriented]: boolean;
  protected quadMatch: QuadMatch;
  protected context: ProxyContext;
  constructor(context: ProxyContext, quadMatch: QuadMatch);
```

It is possible that #138 could help, I don't know, haven't tried.

--- a/tsconfig.esm.json
+++ b/tsconfig.esm.json
@@ -1,8 +1,8 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "module": "NodeNext",
-    "moduleResolution": "nodenext",
+    "module": "preserve",
+    "moduleResolution": "bundler",
     "declaration": true,
     "declarationMap": true,
     "isolatedModules": true

